mt-kahypar/partition/refinement/flows/scheduler.cpp

```diff
@@ -70,9 +70,7 @@ bool FlowRefinementScheduler<GraphAndGainTypes>::refineImpl(
   std::atomic<HyperedgeWeight> overall_delta(0);
   utils::Timer& timer = utils::Utilities::instance().getTimer(_context.utility_id);
   tbb::parallel_for(UL(0), _refiner.numAvailableRefiner(), [&](const size_t i) {
```

Comment:
While we're here, fix the type `numAvailableRefiners`

Reply:
The whole class is deleted in the refactoring :)